test/data/testSP.py

```diff
@@ -241,23 +240,23 @@ def testmoments(self):
             for m in moment32:
                 self.assertEquals(m, 0.0)
 
-    @skipForParser('ADF', 'Does not support metadata yet')

```

Comment:
Commit or PR? I'd argue that this is intimately tied to the symmetry detection/testing.

I also think this should be a squash merge at this point.